doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con-action-scripts.adoc

```diff
@@ -0,0 +1,56 @@
+[id='con-action-scripts_{context}']
+= Action scripts
+
+Action scripts are pieces of code that define the `Script` property or the interceptor action of an element. Action scripts can access the global variables, process variables, and predefined variable `kcontext`. kcontext is an instance of the `ProcessContext` interface. For more information about `kcontext` variable, see the `ProcessContext` https://docs.jboss.org/jbpm/v6.4/javadocs/org/kie/api/runtime/process/ProcessContext.html[Javadoc].
+
+Java and MVEL are supported as dialects for action script definitions. MVEL accepts valid Java code and additionally provides support for nested access to parameters. For example, the MVEL call `person.name` is equivalent of Java call `person.getName()`.
+
+.Example action script in Java and MVEL dialects
+
+[source]
+----
+// Java dialect
+System.out.println(person.getName());
+
+// MVEL dialect
+System.out.println(person.name);
+----
+
+You can also use action scripts to view information about process instances. For example, use the following commands to:
+
+* Return the ID of a process instance
```

Comment:
Suggestion - modify leading sentence and add colon to list:

```
You can also use action script commands to return information about process instances. For example:
+
* To return the ID of a process instance:
+
[source]
----
System.out.println(kcontext.getProcessInstance().getId());
----
```

Reply:
@csherrar, Thanks for catching this. 
Adding colon to the list, however, I think we can keep the list sentences as is to follow the consistency.